### PR TITLE
Remove changeset for publishing

### PR DESCRIPTION
**Pull request by generall**
This PR proposes to replace `changeset` publishing process with simple `pnpm publish`.

There are some reasons:

- `changeset` have overcomplicated release process, which requires full knowledge of the tool even for simplest operations
-  `changeset` works unpredictably: instead on specifying concrete version number we want to release, `changeset` forces us to autoincrement versions. This operation is not idempotent and may create problems, for example, during the CI script problems (performing same operation twice _does_not_ guarantee same result)
- Unclear release strategy: same release script, depending on the obscure conditions, may launch a release or create a commit in some other branch (?)

This PR is WIP as publishing CI pipeline was not tested

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -43,15 +45,8 @@ jobs:
...
-        id: changesets
-        uses: changesets/action@v1
-        with:
-          version: pnpm ci:version
-          publish: pnpm ci:release
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - run: node scripts/prepublish.mjs
+      - run: pnpm publish --access public
```
Comment (Rendez) [inline .github/workflows/release.yaml:49]: package.json already contains this flag.
```json
"publishConfig": {
        "access": "public"
    }
```

Comment (Rendez) [reply .github/workflows/release.yaml:49]: Also Github release will be missing, and so will Changelog.md changes.

**Comment by Rendez** [comment]
The release.yml workflow dispatched on tags means anything pushed to `master` won't run any pipeline, if tags are created off master to issue a release, maybe master should also have a pipeline where we run similar checks as in the pull request workflow, i.e.:
```shell
      - name: Project checks
        run: pnpm -r pre-check

      - name: Integration tests
        run: packages/qdrant-js/scripts/integration-tests.sh
        shell: bash
```

File: .github/workflows/release.yaml (hunk 1 above)
Comment (IvanPleshkov) [reply .github/workflows/release.yaml:49]: In a personal conversation with @generall we decided to skip github releases. It contains changelog and zip sources only, I don't see good reasons to continue to support it

**Pull request merged**